examples/gym/gym_simulator.py

```diff
@@ -0,0 +1,64 @@
+"""
+    A simple robotic simulator to become familiar with the Osgar.
+"""
+
+import gym
+from f110_gym.envs.base_classes import Integrator
+import yaml
+from argparse import Namespace
+import math
```

Comment:
I got used to PEP8 order of imports, i.e. first Python specific (math), then 3rd party (gym, f110) and finally osgar ... all separated by empty lines